js/utils/lib/trumbowyg.js

```diff
@@ -12,26 +13,28 @@ export const editorOptions = {
     'btnGrp-lists',
     ['horizontalRule'],
   ],
+
 };
 
 export function installRichEditor(attachPoint, options = {}) {
-  const opts = {
-    editorOptions: {
-      ...editorOptions,
-      ...options.editorOptions || {},
-    },
-    ...options,
+  const language = app.localSettings.get('language');
+
+  options.editorOptions = {
+    ...defaultEditorOptions,
+    lang: language,
+    ...options.editorOptions || {},
   };
 
+  console.log(`trumboOptions: ${JSON.stringify(options.editorOptions)}`);
```

Comment:
Can we please delete the debugging output?

Reply:
Can we get this into the linter? There's no way we should have listing allow console logs ;D

Reply:
I don't think we need a linting rule against console logs. We do use them in production in a few places.

Reply:
I prefer a no console rule where we override, but if everyone's fine just manually catching them if they slide... ;D